lib/avtp_pipeline/acmp/openavb_acmp_sm_listener.c

```diff
@@ -941,7 +942,18 @@ void openavbAcmpSMListenerSet_talkerTestFastConnect(
 		if (pDescriptor->stream &&
 				pDescriptor->fast_connect_status == OPENAVB_FAST_CONNECT_STATUS_TIMED_OUT &&
 				memcmp(pDescriptor->fast_connect_talker_entity_id, entity_id, 8) == 0) {
+			//
 			// We found a Talker matching the one we have been looking for.
+			//
+
+			// See if it is time to connect yet.
+			struct timespec currenttime;

```

Comment:
I didn't see one, and the code uses "struct timespec" in several other places outside the platform-specific directories, so the change would not be trivial.